astropy/coordinates/builtin_frames/gcrs.py

```diff
@@ -69,5 +68,59 @@ class GCRS(BaseCoordinateFrame):
     obsgeoloc = QuantityFrameAttribute(default=0, unit=u.m, shape=(3,))
     obsgeovel = QuantityFrameAttribute(default=0, unit=u.m/u.s, shape=(3,))
 
-#The "self-transform" is defined in icrs_cirs_transformations.py, because in
-#the current implementation it goes through ICRS (like CIRS)
+# The "self-transform" is defined in icrs_cirs_transformations.py, because in
+# the current implementation it goes through ICRS (like CIRS)
+
+
+class PrecessedGeocentric(BaseCoordinateFrame):
```

Comment:
I don't see any specific tests for this new frame (besides use in constellation code).